lib/core/viewer/components/viewer.component.ts

```diff
@@ -97,6 +97,14 @@ export class ViewerComponent implements OnChanges, OnInit, OnDestroy {
     @Input()
     showToolbar = true;
 
+    /** Hide or show the secondary toolbar */
+    @Input()
+    showSecondaryToolbar = false;
```

Comment:
do we need this new options:
showSecondaryToolbar ?
showImageRotate?

Reply:
We need to be able to hide the rotate/2nd toolbar in case the user has read access but not update access. It is in the ticket requirements. In case the user only has the read access he shouldnt be able to rotate/save the rotation, and same will be with future actions like crop, etc

Reply:
and also even though we are adding more inputs i think with showImageRotate for example we can provide more flexibility if someone will only want to use only crop without the rotate functionality for example. what are your thoughts on this? but showSecondaryToolbar in my opinion is definitely needed in case an user doesn't have update access (also PM requirements)

Reply:
ok thanks for the description of the use cases could we resume maybe in something like readonly?

Reply:
thanks for the suggestion, changed to read-only